tests/UI/pages/BO/advancedParameters/multistore/index.js

```diff
@@ -127,6 +141,26 @@ class MultiStoreSettings extends BOBasePage {
     // Get successful message
     return this.getTextContent(page, this.alertSuccessBlockParagraph);
   }
+
+  /**
+   * Is toggle button visible
+   * @param page
+   * @param row
+   * @returns {Promise<boolean>}
+   */
+  async isToggleButtonVisible(page, row) {
+    return this.elementVisible(page, this.tableColumnActionsToggleButton(row));
+  }
```

Comment:
Can you add more description for this or change the function name to understand :  'Where toggle button is visible, what that means ? '